monitoring-guidelines.md

```diff
@@ -15,42 +22,37 @@ The KubeVirt Users should have the same experience when searching for a node, co
    3.  Network operator metrics should have a `kubevirt_network_` prefix
    4.  Storage operator metrics should have a `kubevirt_cdi_` prefix
    5.  SSP operator metrics should have a `kubevirt_ssp_` prefix
+   6.  HPP Operator metrics should have a `kubevirt_hpp_` prefix
+    For Example, see the following Kubernetes network metrics:
+    - **node**_network_receive_packets_total
+    - **node**_network_transmit_packets_total
+    - **container**_network_receive_packets_total
+    - **container**_network_transmit_packets_total
```

Comment:
This is being rendered as:

```
vi. HPP Operator metrics should have a kubevirt_hpp_ prefix For Example, see the following Kubernetes network metrics:
- node_network_receive_packets_total
- node_network_transmit_packets_total
- container_network_receive_packets_total
- container_network_transmit_packets_total
```

maybe add an empty line before "For example..."

Reply:
Fixed